src/style/page-style.scss

```diff
@@ -192,7 +191,7 @@
     border-radius: $radius;
     box-shadow: $shadow-z200;
     transition: $transition-300;
-    z-index: 100000000;
+    z-index: 2147483647;
```

Comment:
is this the max number for z-index?

Reply:
Yes. When testing the extension on some streaming websites, I noticed that ads show on top of Luca's controls. That's why I maximized the z-index to it's top.